src/dash-table/components/CellFactory.tsx

```diff
@@ -64,7 +64,8 @@ export default class CellFactory {
         const wrappers = this.cellWrappers(
             active_cell,
             columns,
-            viewport.data,
+            virtualized.data,
+            virtualized.offset,
```

Comment:
The offset is now required to reconcile the various memoized handlers, active and selected cell evaluations, etc.